middleware/expvar/expvar.go

```diff
@@ -0,0 +1,42 @@
+package expvar
+
+import (
+	"expvar"
+	"fmt"
+	"net/http"
+
+	"github.com/mholt/caddy/middleware"
+)
+
+type ExpVar struct {
+	Next     middleware.Handler
+	Resource Resource
+}
+
+func (e ExpVar) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error) {
+	if middleware.Path(r.URL.Path).Matches(string(e.Resource)) {
+		expvarHandler(w, r)
+		return http.StatusOK, nil
```

Comment:
Sorry, I'm confused...

Looking at other middleware, they return http.StatusOK, nil after successfully serving a request, which is the case here.